templates/zerver/help/only-allow-admins-to-invite-new-users.md

```diff
@@ -1,6 +1,8 @@
 # Only allow admins to invite new users
-If you are an administrator, you can change the setting to only allow administrators
-to invite new users.
+
+{!admin-only.md!}
+
+You can change the setting to only allow administrators to invite new users.
```

Comment:
This should probably mention that by default, any Zulip user can invite new users.